en/docs/install-and-setup/setup/reference/product-compatibility.md

```diff
@@ -66,11 +66,14 @@ The following is a list of other WSO2 products that is tested with WSO2 API Mana
 <tbody>
 <tr class="even">
 <td>API-M 3.2.0 GA</td>
-<td>GA or WUM update for WSO2 IS-KM-5.10.0</td>
+<td>GA or WUM update for WSO2 IS-5.10.0</td>
 </tr>
 <tr class="even">
 <td>API-M-3.2.0 WUM update</td>
-<td>GA or WUM update for WSO2 IS-KM-5.10.0</td>
+<td>GA or WUM update for WSO2 IS-5.10.0</td>
 </tr>
 </tbody>
 </table>
+
+!!! note 
+    From API Manager 3.2.0 onwards, the prepackaged WSO2 Identity Server as a Key Manager is deprecated and WSO2 Identity Server should be used as the key Manager.
```

Comment:
Theres no deprecation it was removed

Reply:
https://github.com/wso2/docs-apim/pull/2157/commits/20b7bbc6fc82319a9a777cae67836e1280fd9383